Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -150,17 +162,67 @@ theorem mk_zpow (a : G) (n : ℤ) : ((a ^ n : G) : Q) = (a : Q) ^ n :=
 
 @[to_additive (attr := simp)] lemma map_mk'_self : N.map (mk' N) = ⊥ := by aesop
 
+/--
+The subgroup defined by the class of `1` for a congruence relation on a group.
+-/
+@[to_additive
+"The `AddSubgroup` defined by the class of `0` for an additive congruence relation
+on an `AddGroup`."]
+protected def _root_.Con.subgroup (c : Con G) : Subgroup G where
+  carrier := { x | c 1 x }
```

Comment:
I think we should use
```suggestion
  carrier := { x | c x 1 }
```
here to match the `{r : R | t.ringCon r 0}` used for `TwoSidedIdeal`.

Reply:
(or change `TwoSidedIdeal` if you feel it is backwards)